contracts/optimistic-ethereum/OVM/bridge/OVM_L2CrossDomainMessenger.sol

```diff
@@ -17,9 +17,12 @@ import { OVM_BaseCrossDomainMessenger } from "./OVM_BaseCrossDomainMessenger.sol
 
 /**
  * @title OVM_L2CrossDomainMessenger
- * @dev L2 CONTRACT (COMPILED)
- * This contract lives on L2. It sends messages to L1, and relays them from L1.
- */
+ * @dev The L2 Cross Domain Messenger contract sends messages from L2 to L1, and relays messages from L1 onto L1.
+ * 
+ * Compiler used: optimistic-solc
+ * Runtime target: OVM
+ * Execution environment: Layer 2
```

Comment:
I'm a little confused by the difference between these two fields.  To me they feel very analogous.

Is it worth adding a `style-guide.md` somewhere which delineates the standard and explains what the common fields in docs should mean?